styles.scss

```diff
@@ -0,0 +1,248 @@
+  width: 50%;
+  margin-left: 3%;
+  margin-top: 1%;
+
+}
+// .searchbar{
+//  width: 700px;
+// }
+
+
+.padding-20{
+  padding-top: 5px;
+}
+
+// .fb-nav-button-span {
+//   margin-left: 4%;
+// }
+
+.outlined{
+  margin-left: 0;
+  margin-right: 40px;
+  border: 2px solid black;
+}
+
+.outlined-grey{
+  margin-left: 0;
+  margin-right: 40px;
+  border: 2px solid grey;
+}
+
+.fb-nav-buttons {
+  border: $border;
+  margin: 0;
+}
+.fb-nav-buttons-selected{
+  background-color: grey;
+}
+
+.cover-photo{
+  height: 400px;
+  width: 1100px;
+  border: $border;
+}
+
+.edit-profile-anchor{
+  margin-top: 1%;
+  font-weight: underline;
+  color: blue;
+  float: right;
+}
+
+.edit-profile-button{
+  margin-bottom: 1%;
+  margin-top: 3px;
+  font-weight: underline;
+  border: $border;
+  // color: white;
+  // background-color: blue;
+  float: right;
+}
+
+.save-changes-button{
+  font-color: white;
+  border: $border;
+  width: 50%;
+  height: 5%;
+}
+
+.save-changes-field{
+  text-align: center;
+  // width: 80%;
+  padding: 0;
+}
+
+.about-bar{
+  text-align: center;
+  margin-right: 10%;
+  margin-left: 7%;
+  background-color: grey;
+  font-weight: bold;
+  font-size: 2em;
+  color: black;
+  margin-top: 1%;
+  border: $border;
+}
+
+.data-label{
+  margin-left: 10%;
+  float: left;
+  color: black;
+}
+
+.data-value{
+  float: right;
+  margin-right: 70px;
+  color: black;
+}
+.information-panel{
+  margin-left: 8%;
+  margin-right: 10.5%;
+}
+
+.basic-info-h2{
+  margin-left: 10%;
+  color: black;
+}
+
+.profile-photo{
+  margin-top: -250px;
+  margin-left: 40px;
+  height: 200px;
+  width: 200px;
+  position: relative;
+  border: $border;
+}
+
+.profile-username{
+  margin-top: -100px;
+  margin-left: 310px;
+  position: relative;
+  font-weight: bold;
+  color: white;
+}
+
+.photo-container{
+  position:relative;
+}
+
+/*  -----------------------
+Friends page css modifiers
+-------------------------- */
+
+.border-for-friends {
+  border: $border;
+  padding: 10px 10px;
+  margin-top: 10px;
+}
+.border-for-friends:last-child {
+  margin-bottom: 10px;
+}
+
+.bump-right {
+  float: right;
+}
+
+.are-you-code-reviewing {
+  margin-bottom: 50px;
+
+}
+
+.patchwork-class {
+  margin-right: 30px;
+}
+
+/*  -----------------------
+Timeline page css modifiers
+-------------------------- */
+
+.margin-at-top {
+  margin-top: 20px;
+}
+
+.box-me {
+  border: $border;
+  width: 100%;
+}
+
+.col-md-4 img {
+  max-width: 75%;
+  max-height:75%;
+}
+
+.margin-me {
+  margin-bottom:  15px;
+}
+
+.row input {
+  width: 100%;
+  height: 200%;
+}
+
+textarea {
+  margin-left: 15px;
+}
+
+.img-zoom {
+  min-width: 100%;
+  min-height: 100%;
+}
+
+.overlay {
+  bottom: 12%;
+  height: 15%;
+  width: 85%;
+  margin: auto;
+  position: absolute;
+  background-color:#3f3f3f;
+  opacity:0;
+}
+.text_box{
+  weight:bold;
+  font-size:2em;
+  padding:10px;
+  padding-bottom:50%;
+  text-align:center;
+}
+
+.eh:hover .overlay{opacity:0.7;}
+img:hover .img{opacity:0.7;}
```

Comment:
Great use of sass; it's still hard for me to visualize sometimes but perhaps more nesting could have been used, but when in doubt it isn't necessary anyway, it's a skill I'm still trying to improve myself.